src/native/corehost/fxr/command_line.cpp

```diff
@@ -289,10 +289,10 @@ void command_line::print_muxer_info(const pal::string_t &dotnet_root, const pal:
         _X("  Version:      ") _STRINGIFY(HOST_FXR_PKG_VER) _X("\n")
         _X("  Architecture: %s\n")
```

Comment:
```suggestion
        _X("  Architecture: ") _stringify(CURRENT_ARCH_NAME) _X("\n")
```

And then remove the `arch` variable. It seems we don't need `get_current_arch_name()` anymore, and maybe we should replace it with defines everywhere.

Reply:
Will go through for other `get_current_arch_name()` separately.